Demo/Base.lproj/Main.storyboard

```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="17506" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="7AT-KF-rbY">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="19162" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="7AT-KF-rbY">
     <device id="retina6_1" orientation="portrait" appearance="light"/>
     <dependencies>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="17505"/>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="19144"/>
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <scenes>
-        <!--Navigation Controller-->
+        <!--Turbo Navigation Controller-->
         <scene sceneID="ETn-Om-JFy">
             <objects>
-                <navigationController id="7AT-KF-rbY" sceneMemberID="viewController">
+                <navigationController id="7AT-KF-rbY" customClass="TurboNavigationController" customModule="Turbo_Demo" customModuleProvider="target" sceneMemberID="viewController">
```

Comment:
The initial nav controller is instantiated via Storyboard. Its class changed from `UINavigationController` to `TurboNavigationController` which uses the path-configuration for pushing view controllers.